components/engine/src/rocks/util/event_listener.rs

```diff
@@ -64,6 +67,27 @@ impl rocksdb::EventListener for EventListener {
             .inc();
     }
 
+    fn on_background_error(&self, reason: DBBackgroundErrorReason, result: Result<(), String>) {
+        assert!(result.is_err());
+        if let Err(err) = result {
+            CRITICAL_ERROR.with_label_values(&["rocksdb"]).inc();
```

Comment:
maybe set it with label `rocksdb <reason>`, so we can distinguish it with others.